data/json/items/tools.json

```diff
@@ -9482,360 +9482,656 @@
+    "price": 0,
+    "material": ["steel", "plastic"],
+    "weight": 250,
+    "volume": 1,
+    "bashing": 1,
+    "cutting": 0,
+    "to_hit": -2,
+    "max_charges": 2,
+    "initial_charges": 2,
+    "charges_per_use": 0,
+    "turns_per_charge": 1,
+    "ammo": "NULL",
+    "revert_to": "null",
+    "category": "weapons",
+    "use_action": {
+        "type": "explosion",
+        "sound_volume": 5,
+        "sound_msg": "Hsssss.",
+        "no_deactivate_msg": "You've already lit the fuse – get rid of it immediately!",
+        "explosion_power": 1,
+        "fields_radius": 2,
+        "fields_type": "fd_smoke",
+        "fields_min_density": 2,
+        "fields_max_density": 3
+    }    
+  },
+  {
+    "type" : "TOOL",
+    "id" : "flute",
+    "name" : "flute",
+    "name_plural": "flutes",
+    "category" : "tools",
+    "weight" : 250,
+    "color" : "light_gray",
+    "to_hit" : 0,
+    "max_charges": 1,
+    "initial_charges": 1,
+    "charges_per_use": 0,
+    "turns_per_charge": 0,
+    "ammo": "NULL",
+    "use_action": {
+        "type": "musical_instrument",
+        "speed_penalty": 10,
+        "volume": 12,
+        "fun": -5,
+        "fun_bonus": 2,
+        "description_frequency": 100,
+        "descriptions": [
+            "You play a little tune on your flute.",
+            "You play a beautiful piece on your flute.",
+            "You play a piece on your flute that sounds harmonious with nature."
+        ]
+    },
+    "revert_to": "null",
+    "symbol" : "-",
+    "description" : "A simple silver plated flute.",
+    "price" : 5000,
+    "material" : ["iron", "silver"],
+    "volume" : 2,
+    "cutting" : 0,
+    "phase" : "solid",
+    "bashing" : 2
+  },
+  {
+    "type" : "TOOL",
+    "id" : "clarinet",
+    "name" : "clarinet",
+    "name_plural": "clarinets",
+    "category" : "tools",
+    "weight" : 550,
+    "color" : "light_gray",
+    "to_hit" : 1,
+    "max_charges": 1,
+    "initial_charges": 1,
+    "charges_per_use": 0,
+    "turns_per_charge": 0,
+    "ammo": "NULL",
+    "use_action": {
+        "type": "musical_instrument",
+        "speed_penalty": 13,
+        "volume": 14,
+        "fun": -5,
+        "fun_bonus": 2,
+        "description_frequency": 100,
+        "descriptions": [
+            "You play a little tune on your clarinet.",
+            "You play a moving, orchestral piece on your clarinet.",
+            "You play an uplifting marching tune on your clarinet."
+        ]
+    },
+    "revert_to": "null",
+    "symbol" : "|",
+    "description" : "An ornate clarinet made from wood.",
+    "price" : 5500,
+    "material" : ["wood", "iron"],
+    "volume" : 6,
+    "cutting" : 0,
+    "phase" : "solid",
+    "bashing" : 4
+  },
+  {
+    "type" : "TOOL",
+    "id" : "saxophone",
+    "name" : "saxophone",
+    "name_plural": "saxophones",
```

Comment:
I played sax in elementary-school band.  Worn with neck strap, so gonna suggest tool_armor treatment.